modules/local/branch_hits_fasta.nf

```diff
@@ -0,0 +1,39 @@
+process BRANCH_HITS_FASTA {
```

Comment:
As a general thing for all local modules, it's nice to follow the structure of official modules as close as possible (in addition to the contents of the file, which you've already done well):

1. Each module goes in it's own directory, and the script file is called `main.nf`
2. Conda definition is in an associated `environment.yml` file
3. Meta file
4. Tests

1/2 I would recommend doing this for this release, 3/4 can come later as it is a lot of work.